setup.py

```diff
@@ -58,7 +58,7 @@
         'scipy>=1.6',
         # See the following issue for info on the iris pin below:
         # https://github.com/ESMValGroup/ESMValCore/issues/2407
```

Comment:
You might want to update that comment here.

Reply:
Done in 9d5d3f3